audio-echo/app/src/main/cpp/audio_player.cpp

```diff
@@ -179,14 +179,18 @@ AudioPlayer::~AudioPlayer() {
   while (devShadowQueue_->front(&buf)) {
     buf->size_ = 0;
     devShadowQueue_->pop();
-    freeQueue_->push(buf);
+    if(buf != &silentBuf_) {
```

Comment:
that is correct: thank you for the fix.

Reply:
I deleted player queue silent buffer check condition. please check. thanks!